### ISSUE TITLE
Error while exporting a custom animation

### ISSUE THREAD

**Issue by Hancapo** [Psycrow101/io_scene_gta_ifp#6]
I was trying to export an animation without importing an existing one, and I always get this kind of errors:

![image](https://github.com/user-attachments/assets/f2635811-4bdb-41e0-8bb2-2f735cb55b4e)

**Comment by Psycrow101** [comment]
It looks like the bone with the keyframe doesn't have a bon_id assigned
![image](https://github.com/user-attachments/assets/9e57645f-9206-4958-b6c4-5cf76946f9b7)

Can you share the .blend project?

**Comment by Hancapo** [comment]
Yeah, but for some reason is trying to read a bone that doesn't exists in the armature (it might be in the action). Just for the record, this happens A LOT (baking an animation from an IK Rig, retargetting, etc.), All I can do is to edit already existing anims (it still can happen under this context tho), anything else won't work 95% of the time. So, in that regard most of the time is kinda useless.
[not_working.zip](https://github.com/user-attachments/files/16339412/not_working.zip)
